lib/languages/en.php

```diff
@@ -3066,4 +3066,38 @@
     'editCache_orderPicsSaveSuccess' => 'Pics order has been changed',
 
 
+    'vote_criteriaNotPassed' => 'Currently logged user account donesn\'t pass criteria of participation in voting',
```

Comment:
```suggestion
    'vote_criteriaNotPassed' => 'Currently logged user account doesn\'t pass criteria of participation in voting',
```
Just testing how "suggestion" works, I don't remember using it before ;)